processing/data_collection/gazette/spiders/pr_curitiba.py

```diff
@@ -0,0 +1,119 @@
+from dateparser import parse
+import datetime as dt
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+
+class PrCuritibaSpider(scrapy.Spider):
```

Comment:
Inherit from `BaseGazetteSpider`(https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/spiders/base.py#L6) to allow date filtering by item pipelines.